src/video/windows/SDL_windowswindow.c

```diff
@@ -117,6 +124,20 @@ GetWindowStyle(SDL_Window * window)
     return style;
 }
 
+static void
+LoadDWMFunctions()
+{
+    static BOOL triedOnce = FALSE;
+    void *pDwmApiDll = NULL;
+    if (triedOnce)
+        return;
+    triedOnce = TRUE;
+    pDwmApiDll = SDL_LoadObject("dwmapi.dll");
```

Comment:
This will remain loaded after the video subsystem is quit/deinitialized, right? There's a convention in SDL_windowsvideo.c to load and unload referenced winapi DLLs at appropriate times.

Reply:
Yeah, I should change that.

I might be able to just do `SDL_LoadObjact`, `SDL_LoadFunction`, call `DWMSetWindowAttribute`, `SDL_UnloadObject` and the attribute would likely stay applied properly anyway, so no need to leak the handle.